question27/js/login.js

```diff
@@ -0,0 +1,114 @@
+'use strict';
+
+import { Chance } from "chance";
+const chance = new Chance();
```

Comment:
👍

Reply:
I've never heard of Chance.js before!
I see that it can generate random values.
Thanks a lot for the learning.